pom.xml

```diff
@@ -84,8 +64,14 @@
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-validation</artifactId>
+            <groupId>org.springframework.cloud</groupId>
```

Comment:
Denne er nødvendig for at cloud-vault-config skal funke, eller?

Reply:
Jeg tester uten og ser hvor det gjør vondt.